dataclients/kubernetes/routegroup.go

```diff
@@ -593,14 +593,13 @@ func (r *routeGroups) convert(s *clusterState, df defaultFilters) ([]*eskip.Rout
 		}
 
 		// Internal hosts
-		for _, host := range internalHosts {
```

Comment:
Interestingly there is a `continue` statement down the line in the loop that would jump to another iteration of this inner loop while processing of the external hosts would jump to the next iteration of the outer loop. I guess the intention was to continue with outer loop so that is also fixed by this change.